argschema/fields/image.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
```

Comment:
I think as a habit we should implement _serialize functions for all the these Fields, and subclass basic Field types so we get that simple validation for free.  So in this case, it should be a subclassed from mm.fields.Str i think.